lib/pdfkit/pdfkit.rb

```diff
@@ -129,7 +130,8 @@ def normalize_options(options)
       next if !value
 
       # The actual option for wkhtmltopdf
-      normalized_key = "--#{normalize_arg key}"
+      normalized_key = "#{normalize_arg key}"
+      normalized_key = "--#{normalized_key}" unless SPECIAL_OPTIONS.include?(key)
```

Comment:
Many of our users use symbol keys. If I pass in `toc: true` this will break because `:toc` is not in the `SPECIAL_OPTIONS` array, but `'toc'` is. Can you change this to:

```
SPECIAL_OPTIONS.include?(normalized_key)
```